flake8diff/vcs/git.py

```diff
@@ -65,3 +69,7 @@ def changed_files(self):
         return filter(self.filter_file,
                       iter(_execute(' '.join(command))
                            .splitlines()))
+
+    def _is_git_repository(self):
+        return _execute(
+            '{vcs} status'.format(vcs=self.vcs), strict=True, log_errors=False)
```

Comment:
Running flake8-diff on a bare repo is probably going to produce rather confusing results, no matter how hard we try to protect people from themselves. :wink: